ui-src/src/index.html

```diff
@@ -13,7 +13,7 @@
 
 </head>
 
-<body>
+<body onbeforeunload=" return 'Are you really want to perform the action?'">
```

Comment:
This is roughly what I was thinking for this https://stackoverflow.com/a/36763156. It means you could use a modal from ng-bootstrap and it would look a bit nice.